### DISCUSSION TITLE
SecurityContext in Connaisseur deployment

### DISCUSSION THREAD

**Discussion by rkydx** [sse-secure-systems/connaisseur#1244] [Q&A]
In Connaisseur deployment value.yaml can we add ‘fsGroup’ and ‘supplementalGroups’ under the securityContext? I have already OPA Gatekeeper in my Kubernetes cluster and I want to deploy connaisseur. But two constraints of Gatekeeper is blocking connaisseur. Those two constraints are - FSGroup and supplementalGroups. Thats the rationale behind this query.

Thanks

**Comment by hsudbrock** [comment]
I had a similar issue; for that reason I opened an issue and a pull request a week ago (https://github.com/sse-secure-systems/connaisseur/pull/1242) to make this possible.

No reaction so far from the maintainers though.

**Comment by phbelitz** [comment]
should be fixed with https://github.com/sse-secure-systems/connaisseur/pull/1264

**Reply by rkydx** [reply]
If it has been fixed then will there be any update in values.yaml file under helm folder?  I can see the last update to the values.yaml file is 4 months ago.
